documentation/assemblies/assembly-iot-guide.adoc

```diff
@@ -0,0 +1,20 @@
+// This assembly is included in the following assemblies:
+//
+// master-openshift.adoc
+
+:context: {context}-iot
```

Comment:
@ncbaratta, Should this just be "iot"?

Reply:
Depends. Is it going to be a separate book? If not, I don't see a need for a different context at all.

Reply:
I needed new context so that I can include "modules/proc-installing-using-bundle.adoc".

Reply:
Changed title to "Internet of Things (IoT) guide" as we discussed